### PR TITLE
guide: update types of checksums in dvc.lock doc

### PR DESCRIPTION
**Pull request by jorgeorpinel**
per https://github.com/iterative/dvc.org/pull/1494#pullrequestreview-441361208

> You may disregard these recommendations if you used the **Edit on GitHub** button from dvc.org to improve a doc in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This causes GitHub to close it automatically when the PR is merged.

Please choose to [allow us](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork) to edit your branch when creating the PR.

Thank you for the contribution - we'll try to review it as soon as possible. 🙏

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
WIP. Waiting for info in https://github.com/iterative/dvc.org/issues/1469#issuecomment-653797419.

File: content/docs/user-guide/dvc-files-and-directories.md [hunk 1]
```diff
@@ -49,34 +49,41 @@ meta:
...
-  the files or directories tracked with DVC. Typically there is only one per
-  `.dvc` file (but several can be added or combined manually).
-- `deps`: List of <abbr>dependency</abbr> entries for this stage, only present
-  when `dvc import` and `dvc import-url` are used. Typically there is only one
-  (but several can be added manually).
+- `outs` (always present): List of <abbr>output</abbr> entries (details below)
+  that represent the files or directories tracked with DVC. Typically there is
+  only one per `.dvc` file (but several can be added or combined manually).
+- `deps`: List of <abbr>dependency</abbr> entries (details below) for this
+  stage, only present when `dvc import` and `dvc import-url` are used. Typically
```
Comment (shcheklein) [inline content/docs/user-guide/dvc-files-and-directories.md:56]: we should start going away from using stage term in the context of .dvc files

Comment (jorgeorpinel) [reply content/docs/user-guide/dvc-files-and-directories.md]: True. Removed the term (along with a few other small fixes) in 3e8b8829.

File: content/docs/user-guide/dvc-files-and-directories.md [hunk 2]
```diff
@@ -49,34 +49,41 @@ meta:
...
-  `.dvc` file (but several can be added or combined manually).
-- `deps`: List of <abbr>dependency</abbr> entries for this stage, only present
-  when `dvc import` and `dvc import-url` are used. Typically there is only one
-  (but several can be added manually).
-- `wdir` (optional): Working directory for the stage command to run in (relative
-  to the file's location). If this field is not present explicitly, it defaults
-  to `.` (the file's location).
+- `outs` (always present): List of <abbr>output</abbr> entries (details below)
+  that represent the files or directories tracked with DVC. Typically there is
+  only one (but several can be added or combined manually).
```
Comment (jorgeorpinel) [inline content/docs/user-guide/dvc-files-and-directories.md:54]: > Typically there is only one (but several can be added or combined manually).

Is this true for .dvc files? I think only one is present always... Need to check.

Comment (jorgeorpinel) [reply content/docs/user-guide/dvc-files-and-directories.md:54]: Same in `deps` (line 57).

BTW, why not call the fields `out` and `dep` (singular) in that case?

File: content/docs/user-guide/dvc-files-and-directories.md [hunk 3]
```diff
@@ -49,34 +49,41 @@ meta:
...
+- `deps`: List of <abbr>external dependency</abbr> entries (details below). Only
```
Comment (jorgeorpinel) [inline content/docs/user-guide/dvc-files-and-directories.md:55]: deps in .dvc files are always external, right?

File: content/docs/user-guide/dvc-files-and-directories.md [hunk 4]
```diff
@@ -49,34 +49,41 @@ meta:
...
+  present when `dvc import` and `dvc import-url` are used to generate this
+  `.dvc` file. Typically there is only one (but several can be added manually).
+- `wdir` (optional): Working directory for the `outs` and `deps` paths (relative
+  to the `.dvc` file's location). If this field is not present explicitly, it
+  defaults to `.` (the `.dvc` file's location).
```
Comment (jorgeorpinel) [inline content/docs/user-guide/dvc-files-and-directories.md:60]: Do .dvc files even have `wdir` now? I think Ruslan said it's never generated byt DVC commands but you can put it in manually (thus I called it "optional").

File: content/docs/user-guide/dvc-files-and-directories.md (hunk 2 above)
Comment (efiop) [reply content/docs/user-guide/dvc-files-and-directories.md:54]: > Is this true for .dvc files? I think only one is present always... Need to check.

Yes.

> BTW, why not call the fields out and dep (singular) in that case?

Just for historic reasons. We were considering changing the format for those in 1.0 but decided not to. Probably will change in the future.

File: content/docs/user-guide/dvc-files-and-directories.md (hunk 3 above)
Comment (efiop) [reply content/docs/user-guide/dvc-files-and-directories.md:55]: @jorgeorpinel No, import-url works with in-repo files too. Not much product sense in that, but still.

File: content/docs/user-guide/dvc-files-and-directories.md (hunk 4 above)
Comment (efiop) [reply content/docs/user-guide/dvc-files-and-directories.md:60]: Yes, they do.

Comment (efiop) [reply content/docs/user-guide/dvc-files-and-directories.md:60]: Not only by-hand, but with `dvc add -f` too.

File: content/docs/user-guide/dvc-files-and-directories.md (hunk 2 above)
Comment (jorgeorpinel) [reply content/docs/user-guide/dvc-files-and-directories.md:54]: Thanks for the answers Ruslan!

"Yes" this is true? This = "Typically there is only one (but several can be added or combined manually)."

From your other answer I think you mean Yes to This = "I think only one is present always..."

Sorry for the confusing question 😋

Comment (efiop) [reply content/docs/user-guide/dvc-files-and-directories.md:54]: @jorgeorpinel 

"Typically there is only one (but several can be added or combined manually)." is correct.

**Pull request merged**